backend/core/taps/ripe_ris.py

```diff
@@ -29,10 +33,21 @@ def normalize_ripe_ris(msg):
             msg['timestamp'] = float(msg['timestamp'])
         if 'type' in msg:
             del msg['type']
-        if 'source' in msg:
-            source = msg['source']
-            if source in update_to_type:
-                msg['type'] = update_to_type[source]
+        for update_type in update_to_type:
+            if update_type in msg:
+                msg['type'] = update_to_type[update_type]
+                for element in msg[update_type]:
+                    if 'prefixes' in element:
+                        for prefix in element['prefixes']:
+                            try:
+                                if is_subnet_of(str2ip(prefix), conf_prefix):
```

Comment:
FYI: keep in mind that since the "explodePrefixes" parameter is not supported, and a message includes multiple prefixes, we still need to check the prefixes that are actually of interest. So this filtering cannot be avoided.